src/elements/OcDisclosureDrop.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="wrapper" ref="menu" @keyup.esc="closeHandler(true)">
```

Comment:
Let's rename this to some 'oc-' class? To reflect what we're doing with other components.